client/src/SearchByTextPanel.css

```diff
@@ -4,6 +4,6 @@
 }
 
 .search-by-text-textarea {
-  height: 500px;
+  height: 100px;
```

Comment:
@huyen-nguyen the height of the text area for text-based query is reduced to 100px.